web-platform/website/src/styles/Colors.ts

```diff
@@ -1,143 +1,144 @@
-  ColorPink40 = "#ff4aa2",
-  ColorPink50 = "#ff298a",
-  ColorPink60 = "#e31587",
-  ColorPink70 = "#c60084",
-  ColorPink80 = "#7f145b",
-  ColorPink90 = "#50134b",
-  ColorRed05 = "#ffdfe7",
-  ColorRed10 = "#ffbdc5",
-  ColorRed20 = "#ff9aa2",
-  ColorRed30 = "#ff848b",
-  ColorRed40 = "#ff6a75",
-  ColorRed50 = "#ff4f5e",
-  ColorRed60 = "#e22850",
-  ColorRed70 = "#c50042",
-  ColorRed80 = "#810220",
-  ColorRed90 = "#440306",
-  ColorOrange05 = "#fff4de",
-  ColorOrange10 = "#ffd5b2",
-  ColorOrange20 = "#ffb587",
-  ColorOrange30 = "#ffa266",
-  ColorOrange40 = "#ff8a50",
-  ColorOrange50 = "#ff7139",
-  ColorOrange60 = "#e25920",
-  ColorOrange70 = "#cc3d00",
-  ColorOrange80 = "#9e280b",
-  ColorOrange90 = "#7c1504",
-  ColorYellow05 = "#ffffcc",
-  ColorYellow10 = "#ffff98",
-  ColorYellow20 = "#ffea80",
-  ColorYellow30 = "#ffd567",
-  ColorYellow35 = "#f9cd34",
-  ColorYellow40 = "#ffbd4f",
-  ColorYellow50 = "#ffa436",
-  ColorYellow60 = "#e27f2e",
-  ColorYellow70 = "#c45a27",
-  ColorYellow80 = "#a7341f",
-  ColorYellow90 = "#960e18",
-  ColorLightGray05 = "#ffffff",
-  ColorLightGray10 = "#f9f9fb",
-  ColorLightGray20 = "#f0f0f4",
-  ColorLightGray30 = "#e0e0e6",
-  ColorLightGray40 = "#cfcfd8",
-  ColorLightGray50 = "#bfbfc9",
-  ColorLightGray60 = "#afafba",
-  ColorLightGray70 = "#9f9fad",
-  ColorLightGray80 = "#8f8f9e",
-  ColorLightGray90 = "#80808f",
-  ColorLightGray100 = "#DEDEE3",
-  ColorDarkGray05 = "#5b5b66",
-  ColorDarkGray10 = "#52525e",
-  ColorDarkGray20 = "#4a4a55",
-  ColorDarkGray30 = "#42414d",
-  ColorDarkGray40 = "#3a3944",
-  ColorDarkGray50 = "#32313c",
-  ColorDarkGray60 = "#2b2a33",
-  ColorDarkGray70 = "#23222b",
-  ColorDarkGray80 = "#1c1b22",
-  ColorDarkGray90 = "#15141a",
-  ColorMarketingGray10 = "#f9f9fa",
-  ColorMarketingGray20 = "#ededf0",
-  ColorMarketingGray30 = "#cdcdd4",
-  ColorMarketingGray40 = "#b1b1bc",
-  ColorMarketingGray50 = "#9595a2",
-  ColorMarketingGray60 = "#7a7a8b",
-  ColorMarketingGray70 = "#5e5e72",
-  ColorMarketingGray80 = "#42425a",
-  ColorMarketingGray90 = "#1f2033",
-  ColorMarketingGray99 = "#20123a",
-  ColorInk05 = "#393473",
-  ColorInk10 = "#342f6d",
-  ColorInk20 = "#312a64",
-  ColorInk30 = "#2e255d",
-  ColorInk40 = "#2b2156",
-  ColorInk50 = "#291d4f",
-  ColorInk60 = "#271948",
-  ColorInk70 = "#241541",
-  ColorInk80 = "#20123a",
-  ColorInk90 = "#1d1133",
-  ColorBlack = "#000000",
-  ColorWhite = "#ffffff",
-  ColorLink = "#0060df",
-  ColorLinkHover = "#0250bb",
-  ColorLinkVisited = "#b833e1",
-  ColorLinkVisitedHover = "#952bb9",
-  ColorMozNeonBlue = "#00ffff",
-  ColorMozLemonYellow = "#fff44f",
-  ColorMozWarmRed = "#ff4f5e",
-  ColorMozNeonGreen = "#54ffbd",
-  ColorMozDarkPurple = "#6e008b",
-  ColorMozDarkGreen = "#005e5e",
-  ColorMozDarkBlue = "#00458b",
-  ColorMozDarkGray = "#959595",
-  ColorMozLightGray = "#e7e5e2",
-  ColorHelperGray = "#42435A",
-  ColorTransparent = "transparent",
+	ColorGreen05 = "#e3fff3",
```

Comment:
@Carla-Moz btw I think this file needs to be re-saved with the "2 spaces" tabbing

Reply:
GAH!